File: restore_files.py
import os

def restore_file(filepath):
    print(f"Restoring {filepath}...")
    try:
        with open(filepath, 'r', encoding='utf-8') as f:
            content = f.read().strip()
        
        # Split by space and convert each code to a character
        codes = content.split(' ')
        decoded_content = "".join(chr(int(code)) for code in codes if code.isdigit())
        
        with open(filepath, 'w', encoding='utf-8') as f:
            f.write(decoded_content)
        print(f"Successfully restored {filepath}")
    except Exception as e:
        print(f"Error restoring {filepath}: {e}")

files_to_restore = [
    r"c:\xampp\htdocs\naggystore\buytuns\admin.html",
    r"c:\xampp\htdocs\naggystore\buytuns\client.html"
]

for file in files_to_restore:
    if os.path.exists(file):
        restore_file(file)
    else:
        print(f"File not found: {file}")

File: restore_files.php
<?php

function restore_file($filepath) {
    echo "Restoring $filepath...\n";
    if (!file_exists($filepath)) {
        echo "File not found: $filepath\n";
        return;
    }

    $content = trim(file_get_contents($filepath));
    if (empty($content)) {
        echo "File is empty: $filepath\n";
        return;
    }

    $codes = explode(' ', $content);
    $decoded_content = "";
    foreach ($codes as $code) {
        if (is_numeric($code)) {
            $decoded_content .= chr((int)$code);
        }
    }

    if (file_put_contents($filepath, $decoded_content) !== false) {
        echo "Successfully restored $filepath\n";
    } else {
        echo "Error writing to $filepath\n";
    }
}

$files_to_restore = [
    "c:\\xampp\\htdocs\\naggystore\\buytuns\\admin.html",
    "c:\\xampp\\htdocs\\naggystore\\buytuns\\client.html"
];

foreach ($files_to_restore as $file) {
    restore_file($file);
}
?>
